common/common-api/src/main/java/org/ow2/proactive/utils/ObjectByteConverter.java

```diff
@@ -230,6 +231,18 @@ public static Serializable base64StringToSerializable(String input) throws IOExc
         return answer;
     }
 
+    public static Map<String, Serializable> mapOfBase64StringToSerializable(Map<String, String> input)
+            throws IOException, ClassNotFoundException {
+        if (input == null) {
+            return null;
+        }
+        HashMap<String, Serializable> answer = new HashMap<>(input.size());
+        for (Map.Entry<String, String> entry : input.entrySet()) {
+            answer.put(entry.getKey(), base64StringToSerializable(entry.getValue()));
+        }
+        return answer;
```

Comment:
You can replace this by:
```java
return input.entrySet().stream.collect(Collectors.toMap(entry -> entry.getKey(), entry-> base64StringToSerializable(entry.getValue())));
```

Reply:
The upper change can only be done if base64StringToSerializable doesn't throw exception. I think it would be great if the exceptions could be removed from the prototype otherwise forget about my comment.